packages/sns/lib/utils/snsAttributeUtils.ts

```diff
@@ -1,5 +1,19 @@
 import type { ZodSchema } from 'zod'
 
+export function generateTopicSubscriptionPolicy(
+  topicArn: string,
+  supportedSqsQueueNamePrefix: string,
+) {
+  return `{"Version":"2012-10-17","Id":"__default_policy_ID","Statement":[{"Sid":"AllowSQSSubscription","Effect":"Allow","Principal":{"AWS":"*"},"Action":["sns:Subscribe"],"Resource":"${topicArn}","Condition":{"StringLike":{"sns:Endpoint":"${supportedSqsQueueNamePrefix}"}}}]}`
```

Comment:
🟢 We might want to add a constant for `2012-10-17`, but I see it doesn't change often 😆

Reply:
yeah, it's nicer as a constant. added reference to documentation as well